<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Main</string>
    </property>
    <addaction name="actionImport"/>
    <addaction name="actionExport"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>Material</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionRe"/>
    <addaction name="separator"/>
    <addaction name="actionImport_Excel_File"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionFull_List"/>
    <addaction name="actionSearch"/>
    <addaction name="actionSearch_for_Model"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionVersion"/>
   </widget>
   <widget class="QMenu" name="menuProjects">
    <property name="title">
     <string>Projects</string>
    </property>
    <addaction name="actionCurrent_Progress"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menu"/>
   <addaction name="menuProjects"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionImport">
   <property name="text">
    <string>Log In</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Log Out</string>
   </property>
  </action>
  <action name="actionFull_List">
   <property name="text">
    <string>Full List</string>
   </property>
  </action>
  <action name="actionSearch">
   <property name="text">
    <string>Search</string>
   </property>
  </action>
  <action name="actionSearch_for_Model">
   <property name="text">
    <string>Search for Model</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionRe">
   <property name="text">
    <string>Re</string>
   </property>
  </action>
  <action name="actionVersion">
   <property name="text">
    <string>Version</string>
   </property>
  </action>
  <action name="actionCurrent_Progress">
   <property name="text">
    <string>Current Progress</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="toolTip">
    <string>프로그램을 종료합니다.</string>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionImport_Excel_File">
   <property name="text">
    <string>Import Excel File</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
